code/__DEFINES/components.dm

```diff
@@ -36,6 +36,7 @@
 #define COMSIG_COMPONENT_REMOVING "component_removing"			//before a component is removed from a datum because of RemoveComponent: (/datum/component)
 #define COMSIG_PARENT_PREQDELETED "parent_preqdeleted"			//before a datum's Destroy() is called: (force), returning a nonzero value will cancel the qdel operation
 #define COMSIG_PARENT_QDELETED "parent_qdeleted"				//after a datum's Destroy() is called: (force, qdel_hint), at this point none of the other components chose to interrupt qdel and Destroy has been called
+#define COMSIG_TOPIC "handle_topic"                             //generic topic handler (/datum/source, href_list)

```

Comment:
Don't forget to update the comment. Also don't include the source in the comment, all signals have that
```suggestion
#define COMSIG_TOPIC "handle_topic"                             //generic topic handler (usr, href_list)
```